src/test/java/org/arquillian/container/chameleon/ConfigurationTestCase.java

```diff
@@ -80,4 +80,22 @@ public void shouldFailOnMissingContainerFile() throws Exception {
         configuration.setChameleonTarget("wildfly:8.2.0.Final:managed");
         configuration.validate();
     }
+
+    @Test
+    public void shouldReturnFalseForMissingContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("MISSING_TARGET:8.2.0.Final:managed");
+        Assert.assertFalse(isSupported);
+    }
+
+    @Test
+    public void shouldReturnTrueForGivenContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("wildfly:8.2.0.Final:managed");
+        Assert.assertTrue(isSupported);
+    }
+
+    @Test
+    public void shouldReturnTrueForTomcatContainer() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("Tomcat:7.0.47:Remote");
+        Assert.assertTrue(isSupported);
+    }
```

Comment:
Can you add a test of a target/source is invalid formatted like `tomcat:7.0.4` or `tomcat.